### PR TITLE
Fix/update detail header

### PR DESCRIPTION
**Pull request by Tj3rk**

### CODE REVIEW

File: components/full-width-header/full-width-header.less [hunk 1]
```diff
@@ -49,21 +49,36 @@
...
+		flex-direction: row;
+	}
 }
 
 .full-width-header-text-container {
 	display: flex;
 	flex-direction: column;
-	
+	width: 100%;
+	max-width: calc(@column-large-7 - 30);
```
Comment (Tj3rk) [inline components/full-width-header/full-width-header.less]: Does this work without setting px behind 30?

File: components/full-width-header/full-width-header.stories.js [hunk 2]
```diff
@@ -21,5 +21,8 @@ storiesOf('Full width header', module)
 	.addDecorator(withKnobs)
 	.add('full width header with text', () => (
 			<FullWidthHeader
-				header={header} />
+				color={header.color.hex} 
+				title={header.title}
```
Comment (Tj3rk) [inline components/full-width-header/full-width-header.stories.js]: lets use the `text` knob here

File: components/full-width-header/full-width-header.stories.js [hunk 3]
```diff
@@ -21,5 +21,8 @@ storiesOf('Full width header', module)
...
+				authorName={header.author.name}
```
Comment (Tj3rk) [inline components/full-width-header/full-width-header.stories.js]: Same here

File: components/full-width-header/full-width-header.less (hunk 1 above)
Comment (JesseE) [reply components/full-width-header/full-width-header.less]: yes it does, dubbel checked. but for clarity I will add px

**Pull request merged**